examples/example-1/Makefile

```diff
@@ -0,0 +1,26 @@
+define check_string
+	@[ "$$(cat $(2) | grep -o $(1))" = "$(3)" ] && echo "Test passed" || (echo "Expose test for $(1) in $(2) failed" && exit 1)
+endef
+
+define exists
+	$(call check_string,$(1),$(2),$(1))
+endef
+
+define notExists
+	$(call check_string,$(1),$(2),"")
+endef
+
+test-datafiles:
```

Comment:
tests the example project by asserting against the existence and lack of existence of the non-exposed feature keys in generated datafiles.